td/publishing/views.py

```diff
@@ -28,9 +28,14 @@ def resource_language_json(request, kind, lang):
     return JsonResponse(data)
 
 
-def resource_catalog_json(request, kind):
-    resource_type = get_object_or_404(OfficialResourceType, short_name=kind)
-    return JsonResponse(resource_type.data, safe=False)
+def resource_catalog_json(request, kind=None):
+    ret = None
```

Comment:
It doesn't seem necessary to declare `ret` here since you covered both possibilities with the `if...else` below. But it's not like this doesn't work.